cli/cmd/mint.go

```diff
@@ -0,0 +1,114 @@
+package cmd
+
+import (
+	"bytes"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"strconv"
+
+	"github.com/spf13/cobra"
+	"github.com/vulpemventures/nigiri/cli/constants"
+	"github.com/vulpemventures/nigiri/cli/controller"
+)
+
+var MintCmd = &cobra.Command{
+	Use:     "mint <address> <ammount>",
+	Short:   "Generate and send a given quantity of an asset",
+	RunE:    mint,
+	PreRunE: mintChecks,
+}
+
+func mintChecks(cmd *cobra.Command, args []string) error {
+	datadir, _ := cmd.Flags().GetString("datadir")
+
+	ctl, err := controller.NewController()
+	if err != nil {
+		return err
+	}
+
+	if err := ctl.ParseDatadir(datadir); err != nil {
+		return err
+	}
+	if len(args) < 2 {
+		return errors.New("Invalid number of arguments.\nnigiri mint <address> <ammount> [name] [ticker]")
```

Comment:
The helper should be the only place where we document the needed flag. no need to be redundant. Remove 

```suggestion
		return errors.New("Invalid number of arguments.\n")
```

If there is an error the help is printed already

```
Usage:
  nigiri mint <address> <ammount> [flags]
```